src/index.ts

```diff
@@ -1,4 +1,3 @@
 export * from './oas2';
 export * from './oas3';
 export * from './oas';
-export * from './postman';
```

Comment:
Needs change here: https://github.com/stoplightio/prism/blob/master/packages/cli/src/operations.ts#L4, right?

Reply:
Yes, this will be required in Prism